requirements.txt

```diff
@@ -2,7 +2,7 @@ alembic==0.8.2
 APScheduler==3.0.4
 argh==0.26.1
 arrow==0.4.4
-beautifulsoup4==4.3.2
+beautifulsoup4==4.4.0
```

Comment:
Let's factor out this code block and add a test for it: to be sure we're not breaking anything
https://github.com/OpenUpSA/pmg-cms-2/blob/master/pmg/views.py#L135

Let's at least add a second argument to `BeautifulSoup(..., "lxml") since this version of BeautifulSoup adds the following warning:

```
>>> BeautifulSoup("<html><h1>hello</h1><p>bob</p></html>").get_text().strip()
/usr/local/lib/python2.7/site-packages/bs4/__init__.py:166: UserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

To get rid of this warning, change this:

 BeautifulSoup([your markup])

to this:

 BeautifulSoup([your markup], "lxml")

  markup_type=markup_type))
u'hellobob'
```


I've manually verified that this change should be alright and didn't see anything int he changelog indicating that it'll be more of a problem than this warning https://groups.google.com/forum/#!topic/beautifulsoup/Yvj49piFQQY

So I don't think a test of this code is critical and it looks really awkward to test now. If you see a nice way to test it, let's try.